src/twisted/logger/_filter.py

```diff
@@ -136,16 +134,15 @@ class LogLevelFilterPredicate:
     Events that not not have a log level or namespace are also dropped.
     """
 
-    def __init__(self, defaultLogLevel=LogLevel.info):
+    def __init__(self, defaultLogLevel: NamedConstant = LogLevel.info) -> None:
         """
         @param defaultLogLevel: The default minimum log level.
-        @type defaultLogLevel: L{LogLevel}
```

Comment:
In a separate commit, consider updating the coding standard at:

https://github.com/twisted/twisted/blob/trunk/docs/core/development/policy/coding-standard.rst

to remove the use of `@type`, and encouraging the use of type annotations.